### DISCUSSION TITLE
Show what have been included when patching

### DISCUSSION THREAD

**Discussion by SodaWithoutSparkles** [ReVanced/revanced-discussions#615] [Question] [answered]
I sometimes forgot what did I included in the patching stage, which is very useful when debugging/confirming you have actually updated the app, as the version didnt change if you patched the same source APK

Possible location for this in YT is add a "about" tab in "Revanced Spectific Settings", which can include the following:

- Patching method, e.g. `Manager V0.0.14` or `CLI` 
- Patching date, e.g. `2022-09-16`
- Patches included during build time and its versions, might be hidden behind another tab
- Link to Github repo for README, issues, manager, etc
- Link to social media of Revanced, like Discord, Reddit, etc

Which would make debugging much easier

**Comment by Aunali321** [comment, marked as answer]
All of the above feature are already included in manager.

**Reply by SodaWithoutSparkles** [reply]
![Screenshot_2022-09-20-00-24-36-019-edit_app.revanced.manager.flutter.jpg](https://user-images.githubusercontent.com/64138578/191065945-99af68c3-7db8-4075-8856-262cf0cf3acc.jpg)

Where is it?

**Reply by Aunali321** [reply]
Click info

**Reply by SodaWithoutSparkles** [reply]
Maybe also burn that info to the apk, as well as the device ID used to build it. Warn users if the ID mismatches. For example, get the WiFi MAC address.

**Reply by SodaWithoutSparkles** [reply]
> Maybe also burn that info to the apk, as well as the device ID used to build it. Warn users if the ID mismatches. For example, get the WiFi MAC address.

Oh it was there the whole time. How about also burning it to the apk?

**Comment by SodaWithoutSparkles** [comment]
@Aunali321 Does the new app actually update with new set of patches if the version code is the same?

Foe example, I have already patched and installed YT v17.33.42 with patches ABCD, now I patch the same YT v17.33.42 apk again but with patches ACD only. Does this actually installed the one without B? the version is the same

**Reply by Aunali321** [reply]
You can try and check if it works.
